### PR TITLE
[IMP] cli: suggest auto complete script

### PR DESCRIPTION
**Pull request by william-andre**
Add a hint to how to build autocompletion values for odoo-bin

### CODE REVIEW

**Comment by william-andre** [comment]
See discussion on https://github.com/odoo/odoo/pull/64541

File: content/developer/cli.rst [hunk 1]
```diff
@@ -51,6 +51,14 @@ Help & version
 
    shows Odoo version e.g. "Odoo Server 13.0"
 
+.. note:: You can enable auto-completion in your shell by running
```
Comment (Feyensv) [inline content/developer/cli.rst:54]:
```suggestion
.. tip:: You can enable auto-completion in your shell by running
```

?

**Review by Feyensv — COMMENTED** [review]
Interesting and useful information :+1:
Maybe worth adding in 14 + as well ?

**Comment by AntoineVDV** [comment]
@william-andre That actually depends on the page. For this one, 16.0 is good enough because it's very unlikely that we update the "Help" section in an earlier version. For other pages, like Installing Odoo, it's not a good idea to target 16.0 because any (required) change in 14.0/15.0 will conflict at the forward-port.

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Review by Feyensv — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**